README.md

```diff
@@ -544,6 +544,37 @@ api.render(
     403
 ```
 
+## Using the Batch API
+
+Sendwithus' API has a [batch endpoint](https://support.sendwithus.com/api/#batchapi) that can take multiple requests and execute them concurrently. 
+*Note* that Sendwithus recommends keeping batch requests to 10 items or less.
+
+### Creating and Executing a Batch Request
+
+```python
+api = sendwithus.api(API_KEY, DEBUG=False)
+batch = api.start_batch()
+
+emails = [
+    'user+1@example.com',
+    'user+2@example.com',
+    # ...
+    'user+10@example.com'
+]
+
+# Generate the batch request
+for email in emails:
+    batch.send(
+        email_id='TEMPLATE_ID',
+        recipient={
+            'address': emails
```

Comment:
I do believe you may want email here, instead of emails, if I am not mistaken.

Reply:
Good catch!